src/sentry/api/endpoints/account_appearance_details.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+
+import pytz
+from django.conf import settings
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import serializers
+from django.utils.translation import ugettext_lazy as _
+
+from sentry.api.base import Endpoint
+from sentry.constants import LANGUAGES
+from sentry.models import UserOption
+
+
+def _get_timezone_choices():
+    results = []
+    for tz in pytz.common_timezones:
+        now = datetime.now(pytz.timezone(tz))
```

Comment:
do you feel confident these results map correctly to what the frontend can send?

Reply:
Yes. (This also relates to your comment about moment-tz) The data file on frontend is generated by this.

Reply:
ah, ok